app/components/BalanceCell/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+*
+* BalanceCell
+*
+*/
+
+import React from 'react';
+
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import styles from './styles.css';
+
+class BalanceCell extends React.Component { // eslint-disable-line react/prefer-stateless-function
```

Comment:
This is true for all the classes you've created, please change them appropriately.